### PR TITLE
shorten long names

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A   |
|---          |--- |
|Bug Fix?     |no  |
|New Feature? |no  |
|BC Breaks?   |no  |
|Deprecations?|no  |
|Tests Pass?  |yes |
|Fixed Tickets|#660|
|License      |MIT |

### CODE REVIEW

File: src/Elcodi/Bundle/AttributeBundle/DependencyInjection/ElcodiAttributeExtension.php [hunk 1]
```diff
@@ -77,15 +77,15 @@ protected function getConfigurationInstance()
     protected function getParametrizationValues(array $config)
     {
         return [
-            "elcodi.core.attribute.entity.attribute.class" => $config['mapping']['attribute']['class'],
-            "elcodi.core.attribute.entity.attribute.mapping_file" => $config['mapping']['attribute']['mapping_file'],
-            "elcodi.core.attribute.entity.attribute.manager" => $config['mapping']['attribute']['manager'],
-            "elcodi.core.attribute.entity.attribute.enabled" => $config['mapping']['attribute']['enabled'],
+            "elcodi.entity.attribute.class" => $config['mapping']['attribute']['class'],
```
Comment (javiereguiluz) [inline src/Elcodi/Bundle/AttributeBundle/DependencyInjection/ElcodiAttributeExtension.php]: I find it confusing to call a parameter `['mapping']['attribute']['class']` in PHP and `entity.attribute.class` as parameter.  `entity` and `mapping` are not the same concepts.

Comment (cordoval) [inline src/Elcodi/Bundle/AttributeBundle/DependencyInjection/ElcodiAttributeExtension.php]: i agree too, but that is out of the scope of this PR, should we create another ticket for that? I am finding that we have to pipe PRs else this becomes a nightmare of work :+1:

**Comment by cordoval** [comment]
This i hope passes the tests :older_man: I will keep watching over travis and fix accordingly.

**Comment by mmoreram** [comment]
@cordoval Please, can you squash?

Squash

**Comment by cordoval** [comment]
bump @mmoreram

**Pull request merged**